cheroot/test/test_ssl.py

```diff
@@ -166,6 +167,43 @@ def tls_certificate_private_key_pem_path(tls_certificate):
         yield cert_key_pem
 
 
+def _thread_except_hook(exceptions, args):
+    """Append uncaught exception ``args`` in threads to ``exceptions``."""
+    if args.exc_type == SystemExit:
```

Comment:
wouldn't it be more convenient to check the inheritance?
```suggestion
    if issubclass(args.exc_type, SystemExit):
```